### PR TITLE
Disable loopProtect for non-JavaScript code

### LINKED ISSUE #12683: loopProtect triggers in HTML
Challenge [add-an-accessible-date-picker](http://beta.freecodecamp.com/en/challenges/applied-accessibility/add-an-accessible-date-picker) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

The code result includes additional lines of code
Browser: Google Chrome
bugfcc

```html
    
    
<body>
<header>
  <h1>Tournaments</h1>
</header>
<main>
  <section>
    <h2>Mortal Combat Tournament Survey</h2>
    <form>
      <p>Tell us the best date for the competition</p>
      <label for="pickdate">Preferred Date (MM-DD-YYYY):</label>
      
      <!-- Add your code below this line -->
      <label for="date">Enter date (MM-DD-YYYY):</label>
      <input type="date" id="pickdate" name="date">
      
      <!-- Add your code above this line -->
      
      <input type="submit" name="submit" value="Submit">
    </form>
  </section>
</main>
<footer>&copy; 2016 Camper Cat</footer>
</body>

    ```

### PR DESCRIPTION
**Pull request by ghost**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #12683 

#### Description

Workaround for loopProtect triggering in HTML code: disable it for pure markup that contains no `<script>` tags.

### CODE REVIEW

File: client/rechallenge/transformers.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ const transformersForHtmlJS = {
     {
       name: 'add-loop-protect',
       transformer: function addLoopProtect(file) {
+        const _contents = file.contents.toLowerCase();
+        if (file.ext === 'html' && _contents.indexOf('<script>') === -1) {
```
Comment (Greenheart) [inline client/rechallenge/transformers.js]: I think 
```js
_contents.includes('<script>')
```

is cleaner than using `.indexOf()`. It's not a huge difference but it improves readability.

**Review by Greenheart — CHANGES_REQUESTED** [review]
@leonfeng Thanks for fixing this! I can confirm that this solves the issue.

I've just got a small suggestion for how we could improve readability before we merge this :smile:

**Comment by leonfeng** [comment]
@Greenheart Learned something new~ I didn't even know `str.includes()` existed…

**Review by Greenheart — APPROVED** [review]
@leonfeng ES6 goodness! :smile:

Thanks again!

**Comment by systimotic** [comment]
Thanks for fixing this @leonfeng!

**Pull request merged**